src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,31 @@
-export const Tabs = () => {};
+export const Tabs = ({ tabs, activeTabId, onTabSelected, displayTab }) => {
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => (
+            <li
+              key={tab.id}
+              className={tab.id === activeTabId ? 'is-active' : undefined}
+              data-cy="Tab"
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={
+                  () => onTabSelected(tab.id)
```

Comment:
My fault here, the test was asking for your implementation, use as before:
`tab.id === displayTab.id ? undefined : () => onTabSelected(tab.id)`